frontend/src/component/releases/ReleasePlanTemplate/CreateReleasePlanTemplate.tsx

```diff
@@ -70,33 +88,69 @@ export const CreateReleasePlanTemplate = () => {
         }
     };
 
+    const onSidebarClose = () => {};
+
     if (!releasePlansEnabled) {
         return null;
     }
 
     return (
-        <>
-            <FormTemplate
-                title='Create release plan template'
-                description='Create a release plan template to make it easier for you and your team to release features.'
+        <FormTemplate
+            title='Create release plan template'
+            documentationIcon={<ReleaseTemplateIcon />}
+            description='Create a release plan template to make it easier for you and your team to release features.'
+        >
+            <StyledForm onSubmit={handleSubmit}>
+                <TemplateForm
+                    name={name}
+                    setName={setName}
+                    description={description}
+                    setDescription={setDescription}
+                    errors={errors}
+                    clearErrors={clearErrors}
+                />
+                <MilestoneList
+                    milestones={milestones}
+                    setAddStrategyOpen={setAddStrategyOpen}
+                    errors={errors}
+                    clearErrors={clearErrors}
+                />
+                <StyledAddMilestoneButton
+                    variant='text'
+                    color='primary'
+                    onClick={(e) => {
+                        e.preventDefault();
+                        setMilestones([
+                            ...milestones,
+                            {
+                                name: `Milestone ${milestones.length + 1}`,
+                                sortOrder: milestones.length,
+                            },
+                        ]);
+                    }}
```

Comment:
Should we extract this to a `addMilestone` method?

Reply:
I suppose we could, especially if we're making this UI reusable by both pages.